src/routes/webhooks/apple.ts

```diff
@@ -233,7 +237,11 @@ const handleNotifcationRequest = async (
     // Don't proceed any further if it's a test notification
     if (notification.notificationType === NotificationTypeV2.TEST) {
       logger.info(
-        { notification, provider: SubscriptionProvider.AppleStoreKit },
+        {
+          environment,
```

Comment:
We should start using child loggers in pino, easy to forget adding stuff https://getpino.io/#/docs/api?id=base-object, well at some point we will..

Reply:
I miss python and its logger 😬